tests/functional/fetchr.test.js

```diff
@@ -163,13 +163,13 @@ describe('client/server integration', () => {
                 meta: { foo: 'bar' },
                 output: { message: 'error' },
                 rawRequest: {
-                    url: '/api/error?returnMeta=true',
+                    url: 'http://localhost:3000/api/error?returnMeta=true',
```

Comment:
fetch require the full host now?

Reply:
No. `rawRequest` here is `fetchr` custom serialization of a whatwg `Request` instance. `Request.url` returns the host and the path concatenated (it doesn't provide a `path` getter).

You can try on your browser:

```js
let req = new Request('/api')
console.log(req.url)
// https://github.com/api

req = new Request('https://yahoo.com/api')
console.log(req.url)
// https://yahoo.com/api
```

Also, since the field name is url, IMO it seems wrong to just set a path.

Reply:
ok good to know, thanks!